luigi/contrib/azureblob.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, account_name=None, account_key=None, sas_token=None, **kwargs
             * `token_credential` - A token credential used to authenticate HTTPS requests. The token value should be updated before its expiration.
         """
         self.options = {"account_name": account_name, "account_key": account_key, "sas_token": sas_token}
+        kwargs["protocol"] = kwargs.get("protocol") or "https"  # Default protocol to https if it's not set
```

Comment:
if using dict.get, this could just be `kwargs.get('protocol', 'https')`

Reply:
✅